src/screens/AccountUtxoList/index.tsx

```diff
@@ -0,0 +1,52 @@
+import { useEffect } from "react";
+import { StyleSheet, View } from "react-native";
+
+import { NavigationProp } from "@react-navigation/native";
+
+import { Layout } from "../../styles";
+import { useAccountsContext } from "../../components/accounts/AccountsContext";
+import { Utxo } from "../../models/Utxo";
+import navUtils from "../../utils/NavUtils";
+
+import SelectedUtxosHeader from "../../components/accounts/SelectedUtxosHeader";
+
+import UtxoItem from "./components/UtxoItem";
+
+interface Props {
+  navigation: NavigationProp<any>;
+}
+
+export default function AccountUtxoListScreen({
+  navigation
+}: Props) {
+  const accountsContext = useAccountsContext();
+  const { currentAccount } = accountsContext;
+  const { utxos } = currentAccount;
+
+  useEffect(() => {
+    navUtils.setHeaderTitle(currentAccount.name, navigation);
+  }, []);
+
+  const outpoint = (u: Utxo) => `${u.txid}:${u.vout}`;
```

Comment:
this is a duplicate function

Reply:
I've exposed and used the function through TransactionBuilderContext for reuse.